Gateway/Http/Client/TransactionCancel.php

```diff
@@ -64,6 +64,12 @@ public function __construct(
         $client->setUsername($webserviceUsername);
         $client->setPassword($webservicePassword);
 
+		$client->setAdyenPaymentSource($this->_adyenHelper->getModuleName(), $this->_adyenHelper->getModuleVersion());
+
+		$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+		$productMetadata = $objectManager->get('Magento\Framework\App\ProductMetadataInterface');
```

Comment:
Let's not use objectManager, best to use dependency injection in the constructor.